sdlib/d/gc/tcache.d

```diff
@@ -8,11 +8,16 @@ import d.gc.slab;
 import d.gc.spec;
 import d.gc.tbin;
 import d.gc.util;
+import d.gc.extent;
 
 import sdc.intrinsics;
 
+import core.stdc.stdio;
+
 ThreadCache threadCache;
 
+extern(C) void __sd_run_finalizer(void* ptr, size_t usedSpace, void* finalizer);
```

Comment:
Once again, this declaration shouldn't be duplicated all over the codebase. Where is the natural place to declare it?